### DISCUSSION TITLE
Delight: Introduce an "Element Help" room and promote it to new users

### DISCUSSION THREAD

**Discussion by daniellekirkwood** [element-hq/element-meta#542] [Ideas]
### Your use case

Related to https://github.com/vector-im/element-web/issues/22068

We could have a room, staffed by employees and other members of the community that new users are able to view and join as a part of onboarding

**Comment by t3chguy** [comment]
Keep in mind this feature may fail for some users given their servers may not be federating, e.g corpo

**Comment by germain-gg** [comment]
Hm to my understanding, this is what "Element Web/Desktop" does? How would this room be different?

**Comment by daniellekirkwood** [comment]
If we already have a room thats great! We could highlight it better so user's know its there
